packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4301,6 +4315,10 @@ export default class Meeting extends StatelessWebexPlugin {
       LoggerProxy.logger.log(`Meeting:index#join --> New correlation id ${this.correlationId}`);
     }
 
+    if (options.correlationId) {
```

Comment:
1. the docs for JoinOptions need to updated to contain correlationId
2. this seems like the wrong place to do it, because we will end up with the log from line 4315 printing the wrong correlation id and also unnecessary call to setCorrelationId in line 4314,

Reply:
1. done.
2. reset for correct position.